lru_redux.gemspec

```diff
@@ -10,9 +10,11 @@ Gem::Specification.new do |spec|
   spec.email         = ["sam.saffron@gmail.com"]
```

Comment:
you can add yourself to authors / emails here, at this point you have added plenty to deserve that :)